### PR TITLE
Upgrade dependencies to no longer fail migrating empty HTML content

### LINKED ISSUE #4567: raw_html_to_json migration fails when product description is empty
### What I'm trying to achieve
I'm running the migrate command.

### Steps to reproduce the problem
1. Have a product without any text in the description
2. Run migrate

### What I expected to happen
I expected the migrations to apply successfully. But it fails on https://github.com/mirumee/saleor/blob/master/saleor/product/migrations/0096_raw_html_to_json.py when processing a product without a description. It fails on the `html_to_draftjs()` call.
I fixed it by modifying the conditions slightly.
```diff
...
-if not description_raw.strip():
+if product.description.strip() and not description_raw.strip():
```

### Trace
```
remote: Traceback (most recent call last):
remote:   File "manage.py", line 10, in <module>
remote:     execute_from_command_line(sys.argv)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
remote:     utility.execute()
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
remote:     self.fetch_command(subcommand).run_from_argv(self.argv)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
remote:     self.execute(*args, **cmd_options)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
remote:     output = self.handle(*args, **options)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/base.py", line 83, in wrapped
remote:     res = handle_func(*args, **kwargs)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/core/management/commands/migrate.py", line 234, in handle
remote:     fake_initial=fake_initial,
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/db/migrations/executor.py", line 117, in migrate
remote:     state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
remote:     state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
remote:     state = migration.apply(state, schema_editor)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/db/migrations/migration.py", line 124, in apply
remote:     operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
remote:   File "/app/saleor/product/migrations/0096_raw_html_to_json.py", line 24, in convert_products_html_to_json
remote:     product.description_json = html_to_draftjs(product.description)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/html_to_draftjs/__init__.py", line 8, in html_to_draftjs
remote:     return SoupConverter(strict=strict).convert(soup).to_dict()
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/html_to_draftjs/converter.py", line 340, in convert
remote:     self.build_block(body)
remote:   File "/app/.heroku/python/lib/python3.7/site-packages/html_to_draftjs/converter.py", line 180, in build_block
remote:     element_name = element.name.lower()
remote: AttributeError: 'NoneType' object has no attribute 'name'
```

**System information**
Operating system: Windows 10 x64
Browser: Firefox 68

### PR DESCRIPTION
**Pull request by NyanKiyoshi**
This fixes the `None` type error from the `html-to-draftjs` dependency error when ran against empty HTML (https://github.com/NyanKiyoshi/html-to-draftjs/pull/5).

Closes #4567.

## Pull rquest Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ django-storages = {extras = ["google"],version = "*"}
 django-templated-email = ">=2.2.0"
 django-versatileimagefield = ">=1.10"
 django-webpack-loader = ">=0.3.0"
-django-elasticsearch-dsl = ">=0.5.0.<0.6.0"
+django-elasticsearch-dsl = ">=0.5.0,<0.6.0"
```
Comment (NyanKiyoshi) [inline Pipfile]: Note for the reviewers: this was making pipenv trying to install the 6.4 version (0.6.4 -> 6.4.2, I assume) that is now depending on `elasticsearch-dsl >=6.4.0` where we explicitly ask for `>=6.0.0,<6.1.0`.

Not sure if we should upgrade it. But for sure, this is not what this PR is for.

Comment (maarcingebala) [reply Pipfile]: Not sure about it as well; I think @cmiacz had a problem with resolving this conflict in Pipenv yesterday.

**Comment by django-queries** [comment]
Here is the report for c9fb48240a8929efbcae34b475a4ec70074965bf (NyanKiyoshi/saleor @ fix/empty-html-migration)
Base comparison is 94406064dd9eb68951c3b8eeb662718f6d856aae.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  add billing address to checkout     	         34	         34	             20
  add shipping to checkout            	          7	          7	              0
  checkout payment charge             	         14	         14	              0
  complete checkout                   	          6	          6	              0
  create checkout                     	         41	         41	             20

# api.benchmark homepage
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  retrieve main menu                  	          5	          5	              0
  retrieve product list               	          4	          4	              0
  retrieve secondary menu             	          5	          5	              0
  retrieve shop                       	          2	          2	              0

# api.benchmark product
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  product details                     	         13	         13	              3

# api.benchmark variant
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  retrieve variant list               	          9	          9	              2
```

</p>
</details>

File: Pipfile (hunk 1 above)
Comment (NyanKiyoshi) [reply Pipfile]: Yes, me too. It is very hard to find the cause, even by using `pipenv graph` to see the dependency graph. You cannot see the one we are trying to upgrade to, only the current one.

**Pull request merged**